### ISSUE TITLE
.tmux.config.local not start with default windows

### ISSUE THREAD

**Issue by ScottONeal** [gpakosz/.tmux#66]
Thanks for this amazing project!

I'm new to tmux and was trying to setup default windows when starting a new session

ie:

```
new -n Git
neww -n Server
neww -n Misc
```

However, when doing `tmux new-session -s devel`, it only loads with the default `bash` window. I have to a `ctrl-b r` which does a reload and instantiates a new tmux session, in order for those defaults windows to show up.

Am I missing something obvious?

Thanks for any assistance!

**Comment by gpakosz** [comment]
Hello @ScottONeal,

tmux reads its configuration once when starting, that's about it.

I'm not sure of what you're asking:
```
new -n Git
neww -n Server
neww -n Misc
```

This "works" or seem to work because the first time tmux is launched, it reads its configuration and you added commands to your `.tmux.conf.local` copy (which I source from .tmux.conf). You're creating a new session, overriding the window name, then create 2 additional windows. Since you created a session manually, tmux doesn't create one for you as it would when there's no `new-session` command in the configuration.

Now I'm not sure about what you're asking with `tmux new-session -s devel`. What should happen? 

It seems that you're looking for a tmux session manager: I don't use one but there are plenty of them in the wild. And that's out of scope this configuration I would say

**Comment by ScottONeal** [comment]
Thanks for the quick reply!

I'm coming from screen and I'm sure I've misarticulated the problem and what I'm trying to do. Essentially, all I want to do is when I start tmux, I want there to be some default tabs, akin to this screenshot I took

<img width="1053" alt="screen shot 2017-04-14 at 11 07 42 am" src="https://cloud.githubusercontent.com/assets/707059/25047089/f6006d30-2102-11e7-9c24-98a89bfcd098.png">

I added lines like these to my `.tmux.config.local`:
```
new -n Git
neww -n Server
neww -n Misc
```

However, when I start `tmux` the windows don't exist. The only way to get the windows to appear is when I do a `ctrl-b r`, as shown in the screenshot. However, that seems to create an entirely new tmux session. I only want one tmux session. Sorry if this isn't the place to post this issue, it only seems to affect me when I'm using this repo however.

**Comment by gpakosz** [comment]
Here is what I tried:

`/tmp/test.conf`
```
new -n Git
neww -n Server
neww -n Misc
```

```
$ tmux -S /tmp/tmuxtest -f /tmp/test.conf attach
```

I believe it does what you're trying to achieve: tmux default sessions contains the `Git`, `Server` and `Misc` windows.

tmux manual says:
>      command [flags]
>                   This specifies one of a set of commands used to control tmux, as described in the following sections.  If no com-
>                   mands are specified, the new-session command is assumed.

So if you do `tmux -S /tmp/tmuxtest -f /tmp/test.conf` without the `attach` command you end up with 2 sessions: the one created when sourcing `test.conf` and another empty one because you didn't provide a command.

Here, I verified adding the `new -n Git` ... commands to my `~/.tmux.conf.local` copy then launching `tmux attach` also works.
